Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -86,6 +86,81 @@
 TIME_IS_MISSING = 'time_is_missing'
 
 
+# =========== Enrich User Mechanism ============
+class UserMappingObject:
+    def __init__(self, service, should_map_user, table_name='splunk_xsoar_users', xsoar_user_column_name='xsoar_user',
+                 splunk_user_column_name='splunk_user'):
+        self.service = service
+        self.should_map = should_map_user
+        self.table_name = table_name
+        self.xsoar_user_column_name = xsoar_user_column_name
+        self.splunk_user_column_name = splunk_user_column_name
+
+    def _get_record(self, col, value_to_search):
+        """ Gets the records with the value found in the relevant column. """
+        return self.service.kvstore[self.table_name].data.query(**{"query": json.dumps({col: value_to_search})})
```

Comment:
I'm confused, why do you use `**` for the created dict? why not provide it as a key?